packages/Core/Function.candy

```diff
@@ -7,9 +7,13 @@ getArgumentCount function :=
   needs (is function)
   ✨.getArgumentCount function
 
-is0 value := ✨.ifElse (is value) { False } { equals (getArgumentCount value) 0 }
-is1 value := ✨.ifElse (is value) { False } { equals (getArgumentCount value) 1 }
-is2 value := ✨.ifElse (is value) { False } { equals (getArgumentCount value) 2 }
-is3 value := ✨.ifElse (is value) { False } { equals (getArgumentCount value) 3 }
-is4 value := ✨.ifElse (is value) { False } { equals (getArgumentCount value) 4 }
-is5 value := ✨.ifElse (is value) { False } { equals (getArgumentCount value) 5 }
+is0 value := ✨.ifElse (is value) { equals (getArgumentCount value) 0 } { False }
+is1 value := ✨.ifElse (is value) { equals (getArgumentCount value) 1 } { False }
+is2 value := ✨.ifElse (is value) { equals (getArgumentCount value) 2 } { False }
+is3 value := ✨.ifElse (is value) { equals (getArgumentCount value) 3 } { False }
+is4 value := ✨.ifElse (is value) { equals (getArgumentCount value) 4 } { False }
+is5 value := ✨.ifElse (is value) { equals (getArgumentCount value) 5 } { False }
+
+call function0 :=
+  needs (is0 function0)
+  ✨.call function0
```

Comment:
Have a look at the last sentence in the (pretty long) PR description, right above the checklist we never use. I want to remove one, but am not sure which one we want to keep

Reply:
Ahh well 😅 I prefer `run` because it better describes the longer-term intention. Like, of course, you _call_ a function, but only in order to _run_ it (and to get the return value, but the side effects of the function execution are probably the most important part here, otherwise the function could be a constant).